Cargo.toml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[workspace]
+resolver = "2"
+members = [
+    "up-streamer",
+    "examples/simple-example",
+]
+
+[workspace.package]
+rust-version = "1.66.1"
+version = "0.1.5-dev" # uProtocol version
+repository = "https://github.com/eclipse-uprotocol/up-streamer-rust"
+homepage = "https://github.com/eclipse-uprotocol"
+authors = [
+    "Pete LeVasseur <plevasseur@gmail.com>",
```

Comment:
Via @sophokles73:

> Is this supposed to become an exhaustive list of contributors? The Eclipse Foundation considers maintaining such lists obsolete because everybody can derive it from the Git log. This might have some value for pointing out a prime contact for the project, though, if that is what you had intended to do :-)

Reply:
Via @PLeVasseur:
> Huh. I wonder -- should we have one or two primary points of contact for each of the projects that we put in here?
> 
> I get what you mean about keeping track of all contributors to be not needed.

Reply:
ping @sophokles73

Reply:
FMPOV we can simply omit the property. But if you want to keep it with your name in it, that's also fine with me.